target/linux/ipq40xx/files-4.19/arch/arm/boot/dts/qcom-ipq4018-m018u0-s1.dts

```diff
@@ -0,0 +1,76 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "qcom-ipq4018-m018u0.dtsi"
+
+/ {
+	model = "Corewav-M018U0-S1";
+	compatible = "corewav,m018u0-s1";
+
+	memory {
+		device_type = "memory";
+		reg = <0x80000000 0x10000000>;
+	};
+};
+
+&blsp1_spi1 {
+	mx25l25635f@0 {
+		compatible = "mx25l25635f", "jedec,spi-nor";
+		#address-cells = <1>;
+		#size-cells = <1>;
+		reg = <0>;
+		spi-max-frequency = <24000000>;
+
+		SBL1@0 {
+			label = "SBL1";
+			reg = <0x0 0x40000>;
+			read-only;
+		};
```

Comment:
Add empty lines between nodes as you do in the DTSI.

Reply:
Changed